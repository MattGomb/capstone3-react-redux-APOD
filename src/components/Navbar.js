import React from 'react';
import SearchBar from './Search';
import style from '../styles/Navbar.module.css';
import ghibli from '../assets/ghibli.png';

function Navbar() {
  return (
    <nav className={style.navContainer}>
      <div className={style.logoHolder}>
        <img src={ghibli} className={style.logo} alt="Ghibli logo" />
        <h1 className={style.title}>Studio Ghibli Movies</h1>
      </div>
      <div className={style.searchContainer}>
        <SearchBar />
      </div>
    </nav>
  );
}

export default Navbar;
