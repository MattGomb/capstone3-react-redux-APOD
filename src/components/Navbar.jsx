import React from 'react';
import ghibli from '../assets/ghibli.png';

function Navbar() {
  return (
    <nav className={style.navContainer}>
      <div className={style.logoHolder}>
        <img src={ghibli} className={style.logo} alt="Ghibli logo" />
        <h1 className={style.title}>NASA&apos;s Astronomy Picture of the Day</h1>
      </div>
    </nav>
  );
}

export default Navbar;
