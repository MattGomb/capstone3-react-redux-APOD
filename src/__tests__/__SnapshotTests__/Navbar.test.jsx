import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import { render } from '@testing-library/react';
import store from '../../redux/configureStore';
import Navbar from '../../components/Navbar';

describe('the Navbar snapshot', () => {
  it('renders correctly', () => {
    const tree = render(
      <Provider store={store}>
        <Router>
          <Navbar />
        </Router>
      </Provider>,
    );
    expect(tree).toMatchSnapshot();
  });
});