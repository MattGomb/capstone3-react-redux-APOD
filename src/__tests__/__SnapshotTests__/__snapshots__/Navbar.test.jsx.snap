// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the Navbar snapshot renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="navContainer"
      >
        <div
          class="logoHolder"
        >
          <img
            alt="Ghibli logo"
            class="logo"
            src="ghibli.png"
          />
          <h1
            class="title"
          >
            NASA's Astronomy Picture of the Day
          </h1>
        </div>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav
      class="navContainer"
    >
      <div
        class="logoHolder"
      >
        <img
          alt="Ghibli logo"
          class="logo"
          src="ghibli.png"
        />
        <h1
          class="title"
        >
          NASA's Astronomy Picture of the Day
        </h1>
      </div>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
